File: Programming Basics/Conditional Statements - Exercises/04.ToyShop.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double vacationCost = double.Parse(Console.ReadLine());
        int jigsawCount = int.Parse(Console.ReadLine());
        int dollsCount = int.Parse(Console.ReadLine());
        int teddybearsCount = int.Parse(Console.ReadLine());
        int minionsCount = int.Parse(Console.ReadLine());
        int trucksCount = int.Parse(Console.ReadLine());

        double totalForToys = jigsawCount * 2.60 + dollsCount * 3 + teddybearsCount * 4.10 +
            minionsCount * 8.20 + trucksCount * 2;

        int toysCount = jigsawCount + dollsCount + teddybearsCount + minionsCount + trucksCount;

        double totalPrice = 0;

        if (toysCount >= 50)
        {
            totalPrice = totalForToys - (totalForToys * 0.25);
        }
        else
        {
            totalPrice = totalForToys;
        }

        totalPrice = totalPrice - (totalPrice * 0.10);

        if (totalPrice >= vacationCost)
        {
            Console.WriteLine($"Yes! {(totalPrice - vacationCost):f2} lv left.");
        }
        else if(totalPrice < vacationCost)
        {
            Console.WriteLine($"Not enough money! {((vacationCost - totalPrice)):f2} lv needed.");
        }
    }
}

File: Programming Basics - Python/Conditional Statements - Exercises/04.ToyShop.py
vacation_price = float(input())
puzzles_quantity = int(input())
dolls_quantity = int(input())
teddy_bears_quantity = int(input())
minions_quantity = int(input())
trucks_quantity = int(input())

total_price = ((puzzles_quantity * 2.60) + (dolls_quantity * 3) + (teddy_bears_quantity * 4.10) +
                (minions_quantity * 8.20) + (trucks_quantity * 2))

total_toys_sold = puzzles_quantity + dolls_quantity + teddy_bears_quantity + minions_quantity + trucks_quantity

if total_toys_sold >= 50:
    total_price -= total_price * 0.25

profit = total_price - (total_price * 0.10)

if vacation_price <= profit:
    print(f'Yes! {profit - vacation_price:.2f} lv left.')

else:
    print(f'Not enough money! {vacation_price - profit:.2f} lv needed.')